terraform/aws/modules/aws_instance/amis.tf

```diff
@@ -49,6 +49,23 @@ data "aws_ami" "rhel_8" {
   owners = ["309956199498"]
 }
 
+# identify ubuntu 14.04 ami
+data "aws_ami" "ubuntu_1404" {
```

Comment:
Customer is running old versions of Chef Server, which require older versions of Ubuntu, so I added this

Reply:
Is this something we would need to run on a regular basis? Or it is only for this customer case?

Reply:
Since I am the first one to add it, it probably would not be needed on a regular basis. But is there a reason to not add it other than potential future maintenance (updating owner id, or search terms, etc.)?

Reply:
Let us not block this work to get merged since we would like other engineers to use it, if we need to add a test into the pipeline we should do it with a very dummy backup that doesn't need to be stored secretly (in vault)